distribution/asf-release-process-guide.md

```diff
@@ -0,0 +1,656 @@
+- [IPMC](https://lists.apache.org/list.html?general@incubator.apache.org:lte=1y:%22%5BVOTE%5D%20Release%20Apache%20Druid%20(incubating)%22)
+
+
+## Final Release
+
+Once a release candidate has passed the incubator PMC vote, you'll need to do the following:
+
+- Close the IPMC vote thread
+- Publish release artifacts to the release SVN repo
+- Publish the staged Maven repo
+- Wait 24 hours to allow the artifacts to propagate across mirror
+- Update the druid.apache.org website
+- Announce the release
+
+### Close the IPMC vote thread
+
+##### Subject
+
+```
+[RESULT] [VOTE] Release Apache Druid (incubating) 0.15.1 [RC2]
+```
+
+##### Body
+
+```
+Hi all,
+
+The vote to release Apache Druid (incubating) 0.15.1 has passed with 3 +1 binding votes:
+
+Justin Mclean
+Julian Hyde
+Furkan KAMACI
+
+and 1 non-binding +1 vote:
+
+William Guo
+
+Vote thread:
+https://lists.apache.org/thread.html/f97a3743e441747552f54dd61148366d808883e33d55cab0d5161770@<general.incubator.apache.org>
+
+Thank you to the above IPMC members for taking the time to review and provide guidance on our release!
+
+We will proceed with publishing the approved artifacts and sending out the appropriate announcements in the coming days.
+
+On behalf of the Apache Druid Community,
+Clint
+```
+### Create git tag
+
+Tag the rc that passed vote as the release tag and push to github.
+
+```
+$ git checkout druid-0.15.1-incubating-rc2
+$ git tag druid-0.15.1-incubating
+$ git push origin/druid-0.15.1-incubating
+```
+
+### Publish release artifacts to SVN
+
+The final release artifacts are kept in the following repo (same as KEYS):
+
+```
+svn checkout https://dist.apache.org/repos/dist/release/incubator/druid
+```
+
+Create a new directory for the release and put the artifacts there.
+
+```
+$ svn add 0.15.1-incubating
+...
+$ svn commit -m 'add 0.15.1-incubating artifacts'
+```
+
+### Publish the staged Maven repo
+Returning to the staged repo you created for the IPMC vote ( https://repository.apache.org/#stagingRepositories), "Release" the repo to publish the Maven artifacts.
+
+https://central.sonatype.org/pages/releasing-the-deployment.html#close-and-drop-or-release-your-staging-repository
+
+
+### Wait 24 hours
+
+Apache policy requires projects to wait at least 24 hours after uploading artifacts before announcing a release, to allow time for the release artifacts to propagate across mirrors.
+
+http://www.apache.org/legal/release-policy.html#release-announcements
+
+
+### Update druid.apache.org
+
+1. Pull https://github.com/apache/incubator-druid-website and https://github.com/apache/incubator-druid-website-src. These repositories should be in the same directory as your Druid repository that should have the release tag checked out.
+
+
+2. Delete the 'latest' docs from the root of the website src repo (https://github.com/apache/incubator-druid-website-src).
+
+```
+rm -rf docs/latest/*
```

Comment:
I think we should make scripts/build-to-docs script `rm -rf` the target so you could skip this step.

Reply:
sgtm, will do :+1: